packages/pwa-kit-create-app/assets/templates/chakra-storefront/src/components/_app-config/index.jsx.hbs

```diff
@@ -44,33 +40,55 @@ const AppConfig = ({children, locals = {}}) => {
     }
 
     const commerceApiConfig = locals.appConfig.commerceAPI
-
     const appOrigin = useAppOrigin()
-
     const passwordlessCallback = locals.appConfig.login?.passwordless?.callbackURI
 
+    const memoizedHeaders = useMemo(
+        () => headers,
+        [
+            /* dependencies */
+        ]
+    )
+    const memoizedLogger = useMemo(() => createLogger({packageName: 'commerce-sdk-react'}), [])
+
+    const siteId = useMemo(() => locals.site?.id, [locals.site?.id])
+    const locale = useMemo(() => locals.locale?.id, [locals.locale?.id])
+    const currency = useMemo(
+        () => locals.locale?.preferredCurrency,
+        [locals.locale?.preferredCurrency]
+    )
+    const proxy = useMemo(
+        () => `${appOrigin}${commerceApiConfig.proxyPath}`,
+        [appOrigin, commerceApiConfig.proxyPath]
+    )
+    const redirectURI = useMemo(() => `${appOrigin}/callback`, [appOrigin])
+    const passwordlessLoginCallbackURI = useMemo(() => passwordlessCallback, [passwordlessCallback])
+    const defaultDnt = useMemo(() => locals.appConfig.dnt, [locals.appConfig.dnt])
+
     return (
         <CommerceApiProvider
             shortCode={commerceApiConfig.parameters.shortCode}
             clientId={commerceApiConfig.parameters.clientId}
             organizationId={commerceApiConfig.parameters.organizationId}
-            siteId={locals.site?.id}
-            locale={locals.locale?.id}
-            currency={locals.locale?.preferredCurrency}
-            redirectURI={`${appOrigin}/callback`}
-            proxy={`${appOrigin}${commerceApiConfig.proxyPath}`}
-            headers={headers}
-            defaultDnt={DEFAULT_DNT_STATE}
-            logger={createLogger({packageName: 'commerce-sdk-react'})}
-            passwordlessLoginCallbackURI={passwordlessCallback}
-            {{#if answers.project.commerce.isSlasPrivate}}
-            // Set 'enablePWAKitPrivateClient' to true use SLAS private client login flows.
+            siteId={siteId}
+            locale={locale}
+            currency={currency}
+            redirectURI={redirectURI}
+            passwordlessLoginCallbackURI={passwordlessLoginCallbackURI}
+            proxy={proxy}
+            headers={memoizedHeaders}
+            defaultDnt={defaultDnt}
+            logger={memoizedLogger}
+            // Set 'enablePWAKitPrivateClient' to true to use SLAS private client login flows.
             // Make sure to also enable useSLASPrivateClient in ssr.js when enabling this setting.
+            {{#if answers.project.commerce.isSlasPrivate}}
             enablePWAKitPrivateClient={true}
+            {{else}}
+            // enablePWAKitPrivateClient={true}
             {{/if}}
         >
             <MultiSiteProvider site={locals.site} locale={locals.locale} buildUrl={locals.buildUrl}>
-                <ChakraProvider theme={theme}>{children}</ChakraProvider>
+                <ChakraProvider value={theme}>{children}</ChakraProvider>
```

Comment:
In Chakra v3, `value` is the correct prop to use here. This change is what fixes the `_config` error. 

But we needed to make sure the entire file (and other files too) is up to date.